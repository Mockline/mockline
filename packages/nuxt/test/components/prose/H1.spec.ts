import { describe, it, expect } from 'vitest'
// @ts-ignore
import { MH1 } from '#components'
import type { TypeOf } from 'zod'
import ComponentRender from "../../component-render";

describe('H1', () => {
  it.each([
    ['basic case', {}]
  ])('renders %s correctly', async (nameOrHtml: string, options: TypeOf<typeof MH1.props>) => {
    const html = await ComponentRender(nameOrHtml, options, MH1)
    expect(html).toMatchSnapshot()
  })
})
