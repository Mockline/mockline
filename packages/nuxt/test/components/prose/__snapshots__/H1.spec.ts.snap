// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`H1 > renders basic case correctly 1`] = `"<h1 class="text-4xl font-bold mb-8 scroll-mt-[var(--ui-header-height)]"></h1>"`;
